app/services/search_service.rb

```diff
@@ -0,0 +1,24 @@
+class SearchService
+
+  def self.search(event: required)
```

Comment:
Unused method argument - `event`. You can also write as `search(*)` if you want the method to accept any arguments but don't care about them.